lms/envs/devstack_docker.py

```diff
@@ -9,13 +9,12 @@
 
 LOGGING['loggers']['tracking']['handlers'] = ['console']
 
-HOST = 'edx.devstack.edxapp:18000'
+HOST = 'edx.devstack.lms:18000'
 SITE_NAME = HOST
 LMS_ROOT_URL = 'http://{}'.format(HOST)
 
 ECOMMERCE_PUBLIC_URL_ROOT = 'http://localhost:18130'
 ECOMMERCE_API_URL = 'http://edx.devstack.ecommerce:18130/api/v2'
-ENTERPRISE_API_URL = 'http://enterprise.example.com/enterprise/api/v1/'
```

Comment:
Enterprise should point to edxapp I believe. CC @mattdrayer What I should have done in the last PR was this:
```
ENTERPRISE_API_URL = 'http://edx.devstack.edxapp:18000/enterprise/api/v1/'
```